Index: Assignment/portfolio-provider/src/main/resources/application-dev.yml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>server:\r\n  port: 8082\r\n\r\n\r\nspring:\r\n  banner:\r\n    location: classpath:banner/banner.txt\r\n\r\n  datasource:\r\n    driver-class-name: com.mysql.cj.jdbc.Driver\r\n    url: jdbc:mysql://localhost:3306/spookify?useUnicode=true&characterEncoding=utf-8&useSSL=false&serverTimezone=UTC\r\n    username: root\r\n    password: 123456\r\n\r\nmybatis:\r\n  mapperLocations: classpath:mapper/*.xml\r\n  configuration:\r\n    map-underscore-to-camel-case: true
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Assignment/portfolio-provider/src/main/resources/application-dev.yml b/Assignment/portfolio-provider/src/main/resources/application-dev.yml
--- a/Assignment/portfolio-provider/src/main/resources/application-dev.yml	(revision f2933fb5d7033222be8093be4d8739b3b5dd3c0a)
+++ b/Assignment/portfolio-provider/src/main/resources/application-dev.yml	(date 1700045796918)
@@ -10,7 +10,7 @@
     driver-class-name: com.mysql.cj.jdbc.Driver
     url: jdbc:mysql://localhost:3306/spookify?useUnicode=true&characterEncoding=utf-8&useSSL=false&serverTimezone=UTC
     username: root
-    password: 123456
+    password: lxd20030213
 
 mybatis:
   mapperLocations: classpath:mapper/*.xml
Index: .idea/spookify.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/spookify.iml b/.idea/spookify.iml
new file mode 100644
--- /dev/null	(date 1700043039642)
+++ b/.idea/spookify.iml	(date 1700043039642)
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="WEB_MODULE" version="4">
+  <component name="NewModuleRootManager">
+    <content url="file://$MODULE_DIR$">
+      <excludeFolder url="file://$MODULE_DIR$/.tmp" />
+      <excludeFolder url="file://$MODULE_DIR$/temp" />
+      <excludeFolder url="file://$MODULE_DIR$/tmp" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
\ No newline at end of file
Index: Assignment/portfolio-provider/src/main/resources/application-sit.yml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>server:\r\n  port: 8083\r\n\r\n\r\nspring:\r\n  banner:\r\n    location: classpath:banner/banner.txt\r\n\r\n  datasource:\r\n    driver-class-name: com.mysql.cj.jdbc.Driver\r\n    url: jdbc:mysql://localhost:3306/spookify?useUnicode=true&characterEncoding=utf-8&useSSL=false&serverTimezone=UTC\r\n    username: root\r\n    password: 123456\r\n\r\nmybatis:\r\n  mapperLocations: classpath:mapper/*.xml\r\n  configuration:\r\n    map-underscore-to-camel-case: true
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Assignment/portfolio-provider/src/main/resources/application-sit.yml b/Assignment/portfolio-provider/src/main/resources/application-sit.yml
--- a/Assignment/portfolio-provider/src/main/resources/application-sit.yml	(revision f2933fb5d7033222be8093be4d8739b3b5dd3c0a)
+++ b/Assignment/portfolio-provider/src/main/resources/application-sit.yml	(date 1700045796926)
@@ -10,7 +10,7 @@
     driver-class-name: com.mysql.cj.jdbc.Driver
     url: jdbc:mysql://localhost:3306/spookify?useUnicode=true&characterEncoding=utf-8&useSSL=false&serverTimezone=UTC
     username: root
-    password: 123456
+    password: lxd20030213
 
 mybatis:
   mapperLocations: classpath:mapper/*.xml
Index: .idea/modules.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/modules.xml b/.idea/modules.xml
new file mode 100644
--- /dev/null	(date 1700043039676)
+++ b/.idea/modules.xml	(date 1700043039676)
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/spookify.iml" filepath="$PROJECT_DIR$/.idea/spookify.iml" />
+    </modules>
+  </component>
+</project>
\ No newline at end of file
Index: Assignment/portfolio-provider/src/main/resources/application.yml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>server:\r\n  port: 8081\r\n\r\n\r\nspring:\r\n  banner:\r\n    location: classpath:banner/banner.txt\r\n\r\n\r\n\r\n  datasource:\r\n    driver-class-name: com.mysql.cj.jdbc.Driver\r\n    url: jdbc:mysql://localhost:3306/spookify?useUnicode=true&characterEncoding=utf-8&useSSL=false&serverTimezone=UTC\r\n    username: root\r\n    password: 123456\r\n\r\nmybatis:\r\n  mapperLocations: classpath:mapper/*.xml\r\n  configuration:\r\n    map-underscore-to-camel-case: true\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Assignment/portfolio-provider/src/main/resources/application.yml b/Assignment/portfolio-provider/src/main/resources/application.yml
--- a/Assignment/portfolio-provider/src/main/resources/application.yml	(revision f2933fb5d7033222be8093be4d8739b3b5dd3c0a)
+++ b/Assignment/portfolio-provider/src/main/resources/application.yml	(date 1700045796935)
@@ -12,7 +12,7 @@
     driver-class-name: com.mysql.cj.jdbc.Driver
     url: jdbc:mysql://localhost:3306/spookify?useUnicode=true&characterEncoding=utf-8&useSSL=false&serverTimezone=UTC
     username: root
-    password: 123456
+    password: lxd20030213
 
 mybatis:
   mapperLocations: classpath:mapper/*.xml
